src/features/singleRight/delegate/ChooseRightsPage/RightsActionBarContent/RightsActionBarContent.tsx

```diff
@@ -61,19 +70,35 @@ export const RightsActionBarContent = ({
         {rights
           .filter((right: Right) => right.delegable === true)
           .map((right: Right, index: number) => {
-            return (
-              <div key={index}>
-                <Chip.Toggle
-                  checkmark
-                  selected={right.checked}
-                  onClick={() => {
-                    toggleRight(serviceIdentifier, right.action);
-                  }}
-                >
-                  {t(`common.${right.action}`)}
-                </Chip.Toggle>
-              </div>
-            );
+            if (Object.values(LocalizedAction).includes(right.action as LocalizedAction)) {
+              return (
+                <div key={index}>
+                  <Chip.Toggle
+                    checkmark
+                    selected={right.checked}
+                    onClick={() => {
+                      toggleRight(serviceIdentifier, right.action);
+                    }}
+                  >
+                    {t(`common.${right.action}`)}
+                  </Chip.Toggle>
+                </div>
+              );
+            } else {
+              return (
+                <div key={index}>
+                  <Chip.Toggle
+                    checkmark
+                    selected={right.checked}
+                    onClick={() => {
+                      toggleRight(serviceIdentifier, right.action);
+                    }}
+                  >
+                    {right.action}
+                  </Chip.Toggle>
+                </div>
```

Comment:
I suggest setting the text to be displayed in a variable first.
Then you won't need the duplicate code of having the Chip-definitions twice
```suggestion
            const actionText = Object.values(LocalizedAction).includes(right.action as LocalizedAction) ? t(`common.${right.action}`) : right.action;
              return (
                <div key={index}>
                  <Chip.Toggle
                    checkmark
                    selected={right.checked}
                    onClick={() => {
                      toggleRight(serviceIdentifier, right.action);
                    }}
                  >
                    {actionText}
                  </Chip.Toggle>
                </div>
              );
```